### PR TITLE
chore(release): publish

### PR DESCRIPTION
**Pull request by samsiegart**
- @agoric/rpc@0.5.0
 - @agoric/ui-components@0.8.0
 - @agoric/web-components@0.11.0

### CODE REVIEW

**Comment by samsiegart** [comment]
@turadg The command in MAINTAINERS.md `yarn lerna version --no-push --conventional-commits` creates a single commit for all packages, is this okay?

File: packages/ui-components/CHANGELOG.md
Comment (turadg) [inline packages/ui-components/CHANGELOG.md:14]: I should have committed this as `chore`. Could be removed from changelog.

File: packages/ui-components/CHANGELOG.md
Comment (turadg) [inline packages/ui-components/CHANGELOG.md:28]: also a chore, not relevant to changelog

File: packages/ui-components/CHANGELOG.md
Comment (turadg) [inline packages/ui-components/CHANGELOG.md:23]: not relevant to changelog

**Review by turadg — APPROVED** [review]
One commit is okay.

Consider cleaning up the changelog before merging.

**Comment by samsiegart** [comment]
Cleaned up changelog as suggested, thanks for review!

**Pull request merged**
